### DISCUSSION TITLE
How to pass --sync-period flag

### DISCUSSION THREAD

**Discussion by kdi-sag** [actions/actions-runner-controller#1143] [Questions]
Hello,

We are running actions-runner-controller in AKS. We managed to get it synchronizing faster by downloading the actions-runner-controller.yaml and editing the default value to 1m, but there should be a better way. These are the commands we are running to start the controller and deployment. Where do we need to pass the --sync-period flag?
```
kubectl apply -f https://github.com/actions-runner-controller/actions-runner-controller/releases/download/v0.20.2/actions-runner-controller.yaml

kubectl set env deploy controller-manager -c manager GITHUB_ENTERPRISE_URL=${URL} --namespace actions-runner-system

kubectl create secret generic controller-manager -n actions-runner-system --from-literal=${TOKEN}

kubectl apply -f organizationdeployment.yaml
```

Thanks in advance!

**Comment by shichengripple001** [comment]
you can edit the controller deployment, i tried, it worked

**Reply by kdi-sag** [reply]
Yes, that is what we did but I don't think they we meaning editing the file but rather passing it as an argument to a command.

**Reply by shichengripple001** [reply]
you can pass to helm chart also. look for syncPeriod
https://github.com/actions-runner-controller/actions-runner-controller/tree/master/charts/actions-runner-controller

**Reply by kdi-sag** [reply]
Thanks! So the only way to pass it is by using helm? It would be great if there was a way to add it using the kubectl or after the installation with a configuration command.
